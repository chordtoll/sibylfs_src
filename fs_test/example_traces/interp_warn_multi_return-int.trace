@type script
open "f1.txt" [O_CREAT;O_RDWR] 0o600
write! (FD 3) "0123456789" 10
pread (FD 3) 100 0
close (FD 3)
unlink "f1.txt"
