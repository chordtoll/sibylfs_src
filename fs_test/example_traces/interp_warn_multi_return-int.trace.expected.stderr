
Warning:      4: pread (FD 3) 100 0
   more than one possible result value
   randomly continuing execution with RV_bytes("")
        possible results:         {RV_bytes("0123456789");RV_bytes("012345678");RV_bytes("01234567");RV_bytes("0123456");RV_bytes("012345");RV_bytes("01234");RV_bytes("0123");RV_bytes("012");RV_bytes("01");RV_bytes("0");RV_bytes("")}
